src/toil/jobStores/fileJobStore.py

```diff
@@ -313,6 +315,8 @@ def _exportFile(self, otherCls, jobStoreFileID, url):
                 self._move_and_linkback(srcPath, destPath)
             else:
                 atomic_copy(srcPath, destPath)
+            if jobStoreFileID.executable:
```

Comment:
Here again you need to handle the case where `jobStoreFileID` is a `str` and is missing the attribute.